pvlib/irradiance.py

```diff
@@ -2175,14 +2188,17 @@ def dirindex(ghi, ghi_clearsky, dni_clear, zenith, times, pressure=101325.,
     ghi : array-like
         Global horizontal irradiance. [Wm⁻²]
 
-    ghi_clearsky : array-like
+    ghi_clear : array-like
         Global horizontal irradiance from clear sky model. [Wm⁻²]
 
+    .. versionchanged:: 0.11.2
+        Renamed from ``ghi_clearsky`` to ``ghi_clear``.
```

Comment:
```suggestion
        .. versionchanged:: 0.11.2
            Renamed from ``ghi_clearsky`` to ``ghi_clear``.
```

Sorry Dax, I meant the `..` needed to be aligned with the "Global".  Right now sphinx seems to be interpreting `versionchanged` as another parameter.  I think this suggested indentation should work (and apply the same to the others, of course), but let's check the RTD build to be sure.

Reply:
Looks like it worked.  Now the others should be changed to match

![image](https://github.com/user-attachments/assets/8c08c10f-4644-4044-97af-cea48d12ce9e)